packages/core/src/Services/logout.js

```diff
@@ -13,6 +13,7 @@ const doLogout = response => {
     removeCookies('affiliate_token', 'affiliate_tracking', 'onfido_token');
     SocketCache.clear();
     sessionStorage.clear();
+    LocalStore.remove('launchModalShown');
```

Comment:
This is still cleared after each logout which acts like a sesionStorage instead, so it should either use sessionStorage, which will clear by itself or do not clear local storage here if it's intended